.travis.yml

```diff
@@ -13,7 +13,12 @@ install:
   - pip install -r requirements-dev.txt
 
   # If we are testing Flake8 3.x, then install it
-  - if test "$FLAKE8_VERSION" = "3"; then pip install flake8==3.0.0b2; fi
+  # Force jobs to 1 as a workaround to avoid the PicklingError
+  # see https://gitlab.com/pycqa/flake8/issues/164
+  - if test "$FLAKE8_VERSION" = "3"; then
+      pip install flake8==3.0.0b2;
+      echo "jobs=1" >> setug.cfg;
```

Comment:
It is only needed in Flake8 3.0.0 and is a workaround. I don't see why we should cripple the normal usage of Flake8 just because of the bug. Also 3.x is still in beta so as soon as the bug has been fixed we can remove this line again. And as it'll be hopefully fixed before it goes stable, we don't have to care about the existence of the bug.